data/human/fleets.txt

```diff
@@ -1837,6 +1837,39 @@ fleet "Small Southern Pirates"
 		"Clipper (Speedy)"
 	variant 1
 		"Modified Argosy"
+	add variant 10
```

Comment:
you shouldnt use add variant since this isnt an event modification
just use variant 10 etc
same thing for all the others.

Reply:
Oh. I forgot to remove it when moving from the plugin.